loveship/components/buttons/IconButton.tsx

```diff
@@ -11,7 +11,7 @@ export interface IconButtonMods {
 
 export type IconButtonProps = IconButtonCoreProps & IconButtonMods;
 
-export const IconButton = createSkinComponent<UuiIconButtonProps, IconButtonProps>(
+export const IconButton = createSkinComponent<IconButtonCoreProps, IconButtonProps>(
```

Comment:
The same issue with first generic as for Badge

Reply:
fixed